pkg/reconciler/githubsource/webhook_client.go

```diff
@@ -51,8 +51,8 @@ func (client gitHubWebhookClient) Create(ctx context.Context, options *webhookOp
 		//This is to support GitHub Enterprise, this might be something like https://github.company.com/api/v3/
 		baseURL, err := url.Parse(alternateGitHubAPIURL)
 		if err != nil {
-			logger.Infof("Failed to create webhook - Error occured parsing githubAPIURL %s, error was %v", alternateGitHubAPIURL, err)
-			return "", fmt.Errorf("Error occured parsing githubAPIURL: %v", err)
+			logger.Infof("failed to create webhook - error occured parsing githubAPIURL %s, error was %v", alternateGitHubAPIURL, err)
```

Comment:
Log messages (as opposed to errors that are returned) should be full sentences with capitalization and punctuation.

Reply:
Modified in latest commit